data-import/src/main/resources/folijet/data-import/global/import-instance-holding-item.feature

```diff
@@ -0,0 +1,544 @@
+Feature: Util feature to import instance, holding, item. Based on FAT-937 scenario steps.
```

Comment:
@RuslanLavrov  Let's rename this file such as "deafult-import.-instance-holding-item.feature". It is needed because there are a lot of selected values inside mapping profiles,  which can be changed in other cases.

Reply:
done